File: backend/app/models/wearable.py
# Wearable data model
class WearableData:
    def __init__(self, **kwargs):
        self.id = kwargs.get("id")
        self.device_id = kwargs.get("device_id")
        self.timestamp = kwargs.get("timestamp")
        self.heart_rate = kwargs.get("heart_rate")
        self.steps = kwargs.get("steps")
        self.calories = kwargs.get("calories")

File: frontend/src/types/wearable.ts
export interface WearableDataItem {
  id: string;
  tenant_id: string;
  device_id: string;
  timestamp: string;
  data: Record<string, unknown>;
  heart_rate?: number;
  steps?: number;
  calories?: number;
}

export interface WearableSummary {
  date?: string;
  steps: number;
  calories: number;
  heart_rate: {
    min: number;
    max: number;
    avg: number;
  };
}
